bin/test

```diff
@@ -122,7 +122,7 @@ git commit -m "first"
 
 if [ "${1-}" == "staging" ] || [ "${HEROKU_EXEC_TEST_ENV-}" == "staging" ]; then
   echo "=== skipping init tests"
-  export HEROKU_EXEC_URL="https://heroku-exec-staging.herokuapp.com/"
```

Comment:
😳

Reply:
Seems like there were a lot of places this needed updating, not surprised one got missed!